src/commands/migrate.json

```
@@ -34,6 +34,7 @@
             "SLOW",
             "WRITE"
         ],
+        "get_dbid_args": "migrateDbIdArgs",
```

So, what this command really does is execute a `SELECT <N>; RESTORE key` on the target node not on the current node. So it shouldn't really matter if the current client has access to the permissions to a given db on the current node. @hpatro would also appreciate your thoughts. 

```suggestion
```

---

There was an idea that we should fail fast if we don’t have access on current node, I think both opinions make sense and are valid, not sure what’s the best way to handle this, I guess most of the users sync ACLs across nodes.
https://github.com/valkey-io/valkey/pull/2309#issuecomment-3538976222

---

For cluster slot migration, users probably sync ACL across nodes in the cluster, but there are other use cases for MIGRATE. Do tools like RediShake and Riot use MIGRATE?

MIGRATE can also take an AUTH argument with username and password for the other node, which may be different to the auth used on the current node.
Maybe it's safest to not assume anything about ACL on other nodes.

---

Yeah, this makes sense, deleted this logic.

---

Didn't realize the transformation. Looks good to be removed.